amazon-kinesis-client/src/test/java/software/amazon/kinesis/application/TestConsumer.java

```diff
@@ -220,4 +258,53 @@ private void deleteResources(StreamExistenceManager streamExistenceManager, Leas
         log.info("-------------Finished deleting resources.----------------");
     }
 
+    @Data
+    @ToString
+    @AllArgsConstructor
+    public static class StreamScaler implements Runnable {
```

Comment:
Must this be `public` or can it be `private`?

Consider reading https://thefinestartist.com/effective-java/13